.github/dependabot.yml

```diff
@@ -1,11 +1,15 @@
 version: 2
 
 updates:
+  # npm should be used for npm and yarn
```

Comment:
What do you mean with this comment? Dependabot updates the `yarn.lock` file so I'm confused enough by this "ecosystem" property. What does it actually do?

Reply:
I guess it tells Dependanbot what tools to use. I was also confused by it, as other tools (like lerna) specify `yarn` when working with the `yarn.lock` file and not `npm`.

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem

Reply:
I think this confused everyone of us now 😅 It's because of this entry on the linked page. They will use yarn but you still have to signal it with `npm` 
![Screen Shot 2022-02-07 at 10 09 04](https://user-images.githubusercontent.com/1701755/152758184-bfaa248e-c1bc-472c-8e0b-2ecf00f2688c.png)

Reply:
I claim this comment does more harm than good. Let's remove it. Whoever want's to know why it says `npm` should refer to the documentation. Right now I don't see any benefit in keeping it. It obviously causes confusion.
```suggestion
```